fic/eri/v1/router_paired_to_gcp_connections/requests.go

```diff
@@ -66,6 +66,17 @@ type RouteFilter struct {
 	Out string `json:"out" required:"true"`
 }
 
+// MED represents MED value
+type MED struct {
+	Out int `json:"out"`
+}
+
+// SourceHAInfo represents Primary/Secondary parameters
+// for Source of Connection.
+type SourceHAInfo struct {
+	MED *MED `json:"med" required:"true"`
```

Comment:
Why is it pointer?

Reply:
DONE

Reply:
because [Router to Port Connection](https://github.com/nttcom/go-fic/blob/master/fic/eri/v1/router_paired_to_port_connections/requests.go#L72) was implemented as a pointer so i did.

Reply:
Router to Port Connection uses pointer to handle omitempty tag. This field is required, so it doesn't have to be a pointer.

Reply:
DONE. Thanks for your helpful comments :)